src/components/znkAudioPlayer/services/audio.provider.js

```diff
@@ -0,0 +1,28 @@
+(function (angular) {
+    'use strict';
+
+    angular.module('znk.infra.znkAudioPlayer').provider('AudioSrv', function () {
+
```

Comment:
i think it better to leave the mobileSrv and networkSrv, cause almost every things needs to know if it mobile or not, so maybe add thos or remove the is mobile and is android checks for now.